Editor/Resources/EditorWindow/GameLiftPlugin.uss

```diff
@@ -0,0 +1,59 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+:root {
+    height: 100%;
+    width: 100%;
+    
+    --color-menu-background: #333333;
+    --color-content-background: #3F3F3F;
+    --color-button-primary: #2C5D87;
```

Comment:
In my mind, primary means either "**default color** used for the situation" or "color to indicate the **primary use case** for the customer", but here it's used for when the button is selected

Reply:
At this point I had identified that across the plugin, what I would call the primary button, or positive action, had this colour. It so happened that the selected tab also shared this colour. Would you expected a different variable to cover the tab? I'm not clear on it

Reply:
Okay, it sounds like we're on the same page for use case of this color, and if we want to split it for tab button selected that's a one line change with common uss files